tools/RAiDER/dem.py

```diff
@@ -19,7 +19,7 @@
 from osgeo import gdal
 from pyproj import Proj
 from shapely.geometry import shape, Polygon
-from dem_stitcher.stitcher import download_dem as download_stitched_dem
```

Comment:
```
try:
    from dem_stitcher.stitcher import stitch_dem as download_stitched_dem
except:
   from dem_stitcher.stitcher import download_dem as download_stitched_dem
```
I believe this will maintain backwards-compatibility with the earlier version of dem-stitcher.